src/Nethermind/Nethermind.Consensus/Producers/PayloadAttributes.cs

```diff
@@ -29,6 +29,14 @@ public class PayloadAttributes
         
         public Address SuggestedFeeRecipient { get; set; }
         
+        /// <summary>
+        /// GasLimit
+        /// </summary>
+        /// <remarks>
+        /// Only used for MEV-Boost
+        /// </remarks>
+        public long? GasLimit { get; set; }
```

Comment:
We should consider validation in ForkchoiceUpdatedHandler. We have validation for timestamps. Otherwise, we could produce invalid blocks

Reply:
I will just set it to null there, as it is only intended to get back from relay.